Update GetPrint doc (#3345)
qgis/QGIS-Documentation: source/docs/user_manual/working_with_ogc/server/services.rst

    ...
    </WMS_Capabilities>

SERVICE
^^^^^^^

This parameter has to be ``WMS``.

REQUEST
^^^^^^^

This parameter has to be ``GetPrint`` for the **GetPrint** request.

TEMPLATE
^^^^^^^^

This parameter can be used to specify the name of a layout template to use for
printing.

FORMAT
^^^^^^

This parameter specifies the format of map image. Available values are:

- ``jpg``
- ``jpeg``
- ``image/jpeg``
- ``png``

> - ``svg``

Reviewer: Should pdf be listed here?

    ...
    </WMS_Capabilities>

SERVICE
^^^^^^^

> This parameter has to be ``WMS`` in case of the **GetMap** request.

Reviewer 2: ```suggestion
This parameter has to be ``WMS``.
```

Reviewer 2: Or have I misunderstood?

REQUEST
^^^^^^^

> This parameter is ``GetPrint`` in case of the **GetPrint** request.

Reviewer 2: ```suggestion
This parameter has to be ``GetPrint`` for the **GetPrint** request.
```

TEMPLATE
^^^^^^^^

> This parameter allows to specify the name of the layout template to use for

Reviewer 2: ```suggestion
This parameter can be used to specify the name of a layout template to use for
```

printing.

FORMAT
^^^^^^

> This parameter may be used to specify the format of map image. Available

Reviewer 2: ```suggestion
This parameter specifies the format of map image. Available
```

Reviewer 2: The parameter is mandatory

values are:

- ``jpg``
- ``jpeg``
- ``image/jpeg``
- ``png``
- ``svg``

If the ``FORMAT`` parameter is different from one of these values, then an
exception is returned.

ATLAS_PK
^^^^^^^^

> This parameter allows to activate the Atlas rendering by indicating which

Reviewer 2: ```suggestion
This parameter allows activatation of Atlas rendering by indicating which
```

Reviewer 2: Yes!

features we want to print. In order to retrieve an atlas with all features,
the ``*`` symbol may be used (according to the maximum number of feature
allowed in the project configuration).

> When ``FORMAT`` is ``pdf``, then we have a PDF document with one feature per

Reviewer 2: ```suggestion
When ``FORMAT`` is ``pdf``, a PDF document with one feature per
```

Reviewer 2: Is that not included in the next suggestion?
("page will be returned")

Reviewer: Actually it confuses me more.
#### 1. For all other formats, only the *first page* is returned.
Is that possible to have a two pages layout on server? If yes, with jpg, I'd lose the second page? Am I right?

#### 2. "When ``FORMAT`` is ``pdf``, a PDF document with *one feature per page* is output. 
For me (i'm not server user but it'd be inline with the desktop) it's more a matter of file(s) per feature than page per feature. What I naively thought would be the behavior is:
"When ``FORMAT`` is ``pdf``,  a single PDF document combining each feature page(s) is output. For all other formats, each feature is returned in separate file(s)." 

The limitation with pdf on server would be that I can not output separate pdf files for each feature.
note: the concerned part is no longer in this PR.

> page. For all other formats, only the first page is returned.

Reviewer 2: ```suggestion
page will be returned. For all other formats, only the first page is returned.
```

mapX:EXTENT
^^^^^^^^^^^

> This parameter allows to give the extent for a layout map item as

Reviewer 2: ```suggestion
This parameter specifies the extent for a layout map item as
```

xmin,ymin,xmax,ymax.

mapX:ROTATION
^^^^^^^^^^^^^

> This parameter allows too indicate the map rotation in degrees.

Reviewer 2: ```suggestion
This parameter specifies the map rotation in degrees.
```

mapX:GRID_INTERVAL_X
^^^^^^^^^^^^^^^^^^^^

> This parameter allows to indicate the grid line density in X direction.

Reviewer 2: ```suggestion
This parameter specifies the grid line density in X direction.
```

mapX:GRID_INTERVAL_Y
^^^^^^^^^^^^^^^^^^^^

> This parameter allows to indicate the grid line density in Y direction.

Reviewer 2: ```suggestion
This parameter specifies the grid line density in the Y direction.
```

mapX:SCALE
^^^^^^^^^^

> This parameter allows to set a map scale to a layout map item. This is useful

Reviewer 2: ```suggestion
This parameter specifies the map scale for a layout map item. This is useful
```

to ensure scale based visibility of layers and labels even if client and server
may have different algorithms to calculate the scale denominator.

mapX:LAYERS
^^^^^^^^^^^

> This parameter allows to specify layers for a specific layout map item. See

Reviewer 2: ```suggestion
This parameter specifies the layers for a layout map item. See
```

:ref:`See GetMap <qgisserver-wms-getmap>`: for more information on this
parameter.

mapX:STYLES
^^^^^^^^^^^

> This parameter allows to specify a layers' styles defined in a specific

Reviewer 2: ```suggestion
This parameter specifies the layers' styles defined in a specific
```
